Makefile

```diff
@@ -90,6 +91,15 @@ image:
 	Dockerfile \
 	.
 
+.PHONY: plugins
```

Comment:
I think this Make recipe name can be a bit verbose. Apart from that rest looks good.